### ISSUE TITLE
Can't run the latest version(5.1.1) container on synology nas

### ISSUE THREAD

**Issue by xzf16** [docker-library/redmine#314]
Hi,

I try to run the latest version(5.1.1) container on synology nas, but it stopped unxpectly, and I get a log like this:

date	stream	content
2024/02/18 14:24:36	stdout	(See full trace by running task with --trace)
2024/02/18 14:24:36	stdout	Tasks: TOP => generate_secret_token => config/initializers/secret_token.rb
2024/02/18 14:24:36	stdout	/usr/local/bundle/gems/rake-13.1.0/exe/rake:27:in <top (required)>
2024/02/18 14:24:36	stdout	/usr/src/redmine/lib/tasks/initializers.rake:5:in block in <top (required)>
2024/02/18 14:24:36	stdout	failed to get urandom
2024/02/18 14:24:36	stdout	rake aborted!
2024/02/18 14:24:21	stdout	The Gemfile's dependencies are satisfied
2024/02/18 14:24:21	stdout	While it's not a problem now, it could cause errors if you change the version of one of them later.
2024/02/18 14:24:21	stdout	Remove any duplicate entries and specify the gem only once.
2024/02/18 14:24:21	stdout	You should probably keep only one of them.
2024/02/18 14:24:21	stdout	Your Gemfile lists the gem puma (>= 0) more than once.
2024/02/18 14:24:16	stdout	
2024/02/18 14:24:16	stdout	*** Using sqlite3 as fallback. ***
2024/02/18 14:24:16	stdout	
2024/02/18 14:24:16	stdout	warning: missing REDMINE_DB_MYSQL, REDMINE_DB_POSTGRES, or REDMINE_DB_SQLSERVER environment variables
2024/02/18 14:24:16	stdout

**Comment by xzf16** [comment]
I have found it's related to another issue #264 , the root cause is the kernel of the operation system is too old.

But because I am using synology's NAS, it means that I can't update the kernel, it's controlled by synology team.

So is there any way for me to run the latest redmine container in my synology NAS?

**Comment by tianon** [comment]
If you're having issues due to your kernel being too old, unfortunately I don't think there's much we can do (except recommend upgrading it).  Perhaps there's a newer version of Synology's OS available?  (or maybe it supports running VMs where you could more directly control the kernel version)
